code/game/machinery/fat_sucker.dm

```diff
@@ -215,5 +215,5 @@
 		return
 	start_at = 100
 	stop_at = 0
-	to_chat(user, "<span class='notice'>You remove the access restrictions and lower the automatic ejection threshold!</span>")
+	balloon_alert(user, "restrictions lifted, automatic ejection threshold lowered")
```

Comment:
Not necessary, but this honestly might look best as two balloon alerts, "automatic ejection threshold lowered" delayed a bit after the other.

Reply:
added